provider/freemobile/freemobile.go

```diff
@@ -0,0 +1,71 @@
+package freemobile
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/messagebird/sachet"
+)
+
+// FreeMobileConfig is the configuration struct for FreeMobile provider
+type FreeMobileConfig struct {
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
+	Url      string `yaml:"url"`
```

Comment:
Url does not change per user and could indeed be hardcoded.

However, giving endpoint to configuration is a common industry practice.  Having it hardcoded forces to go through all the release process if the url changes. The current endpoint has a notional `ltd`. It will probably change when Freemobile will extend to other European countries.

The best approach may be to leave the url in configuration but also provide a default value. Would that be ok with you ?

Reply:
Sounds good!